Sources/Orbit/Support/Alert/AlertContent.swift

```diff
@@ -0,0 +1,187 @@
+import SwiftUI
+
+struct AlertContent<Content: View, Icon: View, Buttons: View>: View {
```

Comment:
As the `AlertContent` is internal, it could use the new `isSubtle` environment key.

Reply:
Done.